schema/schema.cue

```diff
@@ -0,0 +1,55 @@
+package product_taxonomy
+
+#attribute_gid_regex:       "^gid://shopify/Taxonomy/Attribute/\\d+$"
+#attribute_value_gid_regex: "^gid://shopify/Taxonomy/Attribute/\\d+/\\d+$"
+
+#category_gid_pattern: "^gid://shopify/Taxonomy/Category/[a-zA-Z]{2}(-\\d+)*$"
+
+// This file defines and enforces the shape of the data for dist/attributes.json and dist/categories.json
+// There are additional validations handled in validations.cue in this directory but use this to understand
+// the shape of the data.
+
+// Both the categories.json and attributes.json file are imported into this cue package for the purposes
+// of this validation. These keys might all be in one file, or split across multiple, cue doesn't care.
+
+// Present in categories.json / categories_data.cue. If this is specified in multiple files this enforces their equality.
+version!: string & =~"^\\d+.\\d+.\\d+$"
+
+// Present in attributes.json / attributes_data.cue
+attributes!: [
+	...{
+		id!:   string & =~#attribute_gid_regex
+		name!: string
+		values!: [
+			...{
+				id!:   string & =~#attribute_value_gid_regex
```

Comment:
Curious if there was a particular reason you dropped the capture we had before (or maybe that wasn't doing what I was expecting):

```
	id:   =~"^\(#AttributeGid)$"
	// ...
	values: [...{
		id:   =~"^\(#AttributeGid)/\\d+$"
		// ...
	}]

Reply:
Non-blocking and happy to add in in a followup PR. Just curious if there was something I wasn't thinking about here